### PR TITLE
Add New Satellite of Interest

### PR DESCRIPTION
**Pull request by dhruvcheemakurti**
Allows the user to add a new satellite of interest with a user input of name and a valid unique noradID to track.

https://github.com/LowerEarthOrbiters/Lower_Earth_Orbiters/assets/23438377/075c00c0-e006-436b-8844-add337aac9f0

### CODE REVIEW

File: src/leo-client-app/src/components/SatellitesOfInterest.tsx [hunk 1]
```diff
@@ -63,6 +85,19 @@ const SatellitesOfInterest = ({ userId }: Props) => {
     fetchData();
   }, []);
 
+  const addSatellite = async (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    console.log('Satellite Name:', satelliteName, 'NORAD ID:', noradId);
```
Comment (Umang-Rajkarnikar) [inline src/leo-client-app/src/components/SatellitesOfInterest.tsx:90]: remove

File: src/leo-client-app/src/components/SatellitesOfInterest.tsx [hunk 2]
```diff
@@ -63,6 +85,19 @@ const SatellitesOfInterest = ({ userId }: Props) => {
...
+    const newSatellite = {
+      name: satelliteName,
+      noradId,
+      satId: "newId",
+    };
+    setSatellites([...satellites, newSatellite]);
```
Comment (Umang-Rajkarnikar) [inline src/leo-client-app/src/components/SatellitesOfInterest.tsx:96]: We don't need this if we are sending the api req right?

**Comment by quinnha** [comment]
can you make the add new satellite button right aligned, and make the popup dark as well?

File: src/leo-client-app/src/components/SatelliteName.tsx [hunk 3]
```diff
@@ -12,6 +12,7 @@ const SatelliteName = ({ satelliteName }: Props) => {
     <Box
       className="material-themedisplaymedium"
       sx={{
+        color: "white",
```
Comment (quinnha) [inline src/leo-client-app/src/components/SatelliteName.tsx:15]: use var here please

File: src/leo-client-app/src/components/SatellitesOfInterest.tsx [hunk 4]
```diff
@@ -22,7 +33,18 @@ type SatelliteDetails = {
...
+  const [noradId, setNoradId] = useState('');
 
+  const handleClickOpen = () => {
+    setOpen(true);
+  };
+
+  const handleClose = () => {
+    setOpen(false);
+  };
+
```
Comment (quinnha) [inline src/leo-client-app/src/components/SatellitesOfInterest.tsx:47]: can just be handleClick = () => {setOpen(!open)}, or something like that, make it 1 function

**Pull request merged**